pull_request_template.md

```diff
@@ -1,18 +1,9 @@
-## What did you do? 
-- [describe the proposed changes]
-- 
-
-
-## Why did you make this change?
-(Cite Issue number OR provide rationalization of changes if no issue exists)
-(If fixing a bug, please add any relevant error or traceback)
-
+(Please replace this header with a description of your pull request. Please include *BOTH* what you did and why you made the change. The "why" may simply be citing a relevant Galaxy issue.) 
```

Comment:
Now that we are at it, maybe add here a recommendation of including screenshots or gifs for UI components and remove the `For UI Components` section at the end?
If the screenshot is already included ticking the box feels a bit redundant.